composer.json

```diff
@@ -7,14 +7,14 @@
         "symfony/debug": "^2.6|^3.0",
         "symfony/process": "^2.6|^3.0",
         "react/event-loop": "^0.4",
-        "react/stream": "^0.4",
-        "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
-        "react/socket-client": "^0.5.0",
-        "react/socket": "^0.5.0",
+        "react/stream": "^0.6",
+        "react/http": "dev-server-request",
+        "react/socket-client": "^0.6",
+        "react/socket": "^0.5",
```

Comment:
Can we allow ^0.5|^0.6|^0.7 here? This would be in line with what clue proposes for ratchet.